create_table.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+require 'pg'
+require 'dotenv/load'
+
+PG.connect(dbname: ENV['DB_NAME'], user: ENV['DB_USER'], password: ENV['DB_PASSWORD']) do |conn|
+  conn.exec('CREATE TABLE memos (id SERIAL PRIMARY KEY, title VARCHAR(50) NOT NULL, content TEXT)')
```

Comment:
```suggestion
  conn.exec(<<~SQL)
    CREATE TABLE memos (
      id SERIAL PRIMARY KEY,
      title VARCHAR(50) NOT NULL,
      content TEXT
    )
  SQL
```

ヒアドキュメントを使った方が読みやすいかも

Reply:
[SQL文の可読性を上げるために修正](https://github.com/s-tone-gs/sinatra-memoapp/pull/2/commits/24dcd9505be3b433580c9b13a0913d3412c1e9a9)にて反映しました！